### DISCUSSION TITLE
Merging multiple models (with duplicates) into one common model without duplicates

### DISCUSSION THREAD

**Discussion by manojb23** [pgmodeler/pgmodeler#1509] [General]
I have a problem.

I have got three database models in three seperate dbm files created through PG Modeler. 

The tables and relationships are duplicated among these three dbm files.  The developer created them as three seperate dbm files since it is easier to visualize them on the screen than laying out 30 odd tables , each having 30 attributes in one single canvas.

I however have a problem that I cannot deploy this to database. I think it should be one DBM to deploy to database. How do I merge all these three DBM files to single DBM file in PG Modeler? 

Do I have to recreate one from scratch?

**Comment by rkhaotix** [comment]
You have two options:

1) Recreate the model that reunites all the objects from the three models

2) Generate the SQL script of each model and execute them manually.

pgModeler does not support merging models, for now.
